src/main/webapp/college-info.js

```diff
@@ -289,8 +289,61 @@ function createCommentElement(name, message, time) {
 function prepReviewForm(id) {
   const submitReviewForm = document.getElementById('submit-review');
   submitReviewForm.setAttribute('action', `/data?id=${id}`);
+
   const idInputElement = document.getElementById('school-id');
   idInputElement.setAttribute('value', id);
 }
 
+/**
+ * Sets the hidden elements to be shown and the shown elements to be hidden.
+ * @param {HTMLElement} showElement HTML element that is hidden/will be shown.
+ * @param {HTMLElement} hideElement HTML element that is shown/will be hidden.
+ * @param {HTMLElement} showElementNavButton HTML nav button to activate.
+ * @param {HTMLElement} hideElementNavButton HTML nav button to deactivate.
+ */
+function toggleElementsDisplay(showElement, hideElement, showElementNavButton,
```

Comment:
I would change the naming around this away from "elements" and to something like "sections" or similar. This would imply that you're going to look through the DOM for any hidden element and flip its visibility, which isn't the case.